### PR TITLE
Zigbee2mqtt lqi

### PR DESCRIPTION
**Pull request by atrovato**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [ ] If your changes affects code, did your write the tests?
- [ ] Are tests passing? (`npm test` on both front/server)
- [ ] Is the linter passing? (`npm run eslint` on both front/server)
- [ ] Did you run prettier? (`npm run prettier` on both front/server)
- [ ] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Please provide a description of the change here. It's always best with screenshots, so don't hesitate to add some!

### CODE REVIEW

**Comment by atrovato** [comment]
I wait for dashboard display...

**Comment by atrovato** [comment]
It can be insterresting to merge it wirh https://github.com/GladysAssistant/Gladys/pull/1575

File: server/services/zigbee2mqtt/exposes/numericType.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ module.exports = {
     return value;
   },
   readValue: (expose, value) => {
+    if (expose.name === 'lqi') {
```
Comment (VonOx) [inline server/services/zigbee2mqtt/exposes/numericType.js]: Must be `linkquality`

File: server/services/zigbee2mqtt/exposes/numericType.js [hunk 2]
```diff
@@ -139,6 +143,14 @@ module.exports = {
         max: 150,
       },
     },
+    lqi: {
```
Comment (VonOx) [inline server/services/zigbee2mqtt/exposes/numericType.js]: Same, must be `linkquality`

**Review by VonOx — CHANGES_REQUESTED** [review]

**Comment by VonOx** [comment]
I comment here cause not sure if #1546 is involved

image
image

```json
{
          "name": "Signal strength",
          "selector": "zigbee2mqtt-cuisine-signal-integer-linkquality",
          "category": "signal",
          "type": "integer",
          "read_only": true,
          "unit": null,
          "min": 0,
          "max": 255,
          "last_value": 2,
          "last_value_changed": "2022-09-06T13:04:35.969Z"
}
```

image

**Comment by VonOx** [comment]
Not sure why my features was created at 255 max.

After manually set to 5 max it's ok in dashboard.

image

File: server/services/zigbee2mqtt/exposes/numericType.js (hunk 1 above)
Comment (atrovato) [reply server/services/zigbee2mqtt/exposes/numericType.js]: You're right!

File: server/services/zigbee2mqtt/exposes/numericType.js (hunk 2 above)
Comment (atrovato) [reply server/services/zigbee2mqtt/exposes/numericType.js]: You're right!

**Comment by atrovato** [comment]
@VonOx max with 255 was an old version.

**Review by VonOx — DISMISSED** [review]

**Comment by Pierre-Gilles** [comment]
Tests are failing on Z2M discover: 

Screenshot 2022-09-08 at 12 10 31

Screenshot 2022-09-08 at 12 10 36

**Comment by atrovato** [comment]
@VonOx in fact, max=255 is provided by z2m device, I take it from the device if it is provided...
I will need to add a "override" option to force the defined mapping.

**Comment by VonOx** [comment]
OK that's explain the behavior.

Not sure if it's a good way to store a converted value. ( I know we have already discussed this on forum 😉)

**Comment by Pierre-Gilles** [comment]
@VonOx Can you retry on your side and confirm it's working fine? :)

**Comment by VonOx** [comment]
> @VonOx Can you retry on your side and confirm it's working fine? :)

It is, can be merged.

**Review by Pierre-Gilles — APPROVED** [review]
Good for me, good work 👏

**Pull request merged**